libs/utils/executor.py

```diff
@@ -464,38 +473,38 @@ def _wload_init(self, tc, wl_idx):
         wload = self._wload_conf(wl_idx, wlspec)
 
         # Keep track of platform configuration
-        self.te.test_dir = '{}/{}:{}:{}'\
```

Comment:
That's just a cosmetics chunk... can you keep them on a separate patch (maybe before)?